pkgs/os-specific/linux/librist/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub, meson, ninja, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "librist";
+  version = "0.2.6";
+
+  src = fetchFromGitHub {
+    owner = "nanake";
+    repo = "librist";
+    rev = "v${version}";
+    sha256 = "sha256-EgWEMRiBJGq7ILmRqdL7Ok6/FhZNmYgiYyWHUhTZqCA=";
+  };
+
+  mesonFlags = [
+    "--default-library=static"
+  ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  meta = with lib; {
+    description = "A library that can be used to easily add the RIST protocol to your application.";
+    homepage = "https://code.videolan.org/rist/librist";
+    license = with licenses; [ lgpl21Plus gpl2Plus ];
```

Comment:
librist itself uses the BSD 2-clause license: https://code.videolan.org/rist/librist/-/blob/master/COPYING

However, there's a bunch of stuff in the `contrib` directory: https://code.videolan.org/rist/librist/-/tree/master/contrib ... Not sure how much of it ends up in the build, or how it's licensed.